From: Philip Withnall <withnall@endlessm.com>
Date: Mon, 3 Sep 2018 13:04:21 +0100
Subject: build: Drop AC_C_CONST from configure.ac

Any compiler we care about now supports the `const` keyword, so we no
longer need to check for it. autoconf has recommended this macro is
obsolete:

https://www.gnu.org/software/autoconf/manual/autoconf-2.60/html_node/C-Compiler.html#index-AC_005fC_005fCONST-755

Signed-off-by: Philip Withnall <withnall@endlessm.com>

Bug: https://gitlab.gnome.org/GNOME/glib/issues/1313
Origin: upstream, 2.59.0, commit:60adc1efbbc17ef6d86084bc56c265d4ae45bb9e
---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 028a3f8..cd66b0d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -533,8 +533,6 @@ AS_IF([test x$ac_cv_sizeof_long_long = x8], [
 	], [AC_MSG_RESULT(none)])
 ])
 
-AC_C_CONST
-
 dnl
 dnl check in which direction the stack grows
 dnl
